### PR TITLE
Adds Revo LDS demo.

### PR DESCRIPTION
**Pull request by damonkohler**

### CODE REVIEW

File: cartographer_ros/launch/demo_revo_lds.launch [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          -configuration_directory $(find cartographer_ros)/configuration_files
+          -configuration_basename revo_lds.lua"
+      output="screen">
+    <remap from="scan" to="horizontal_laser_2d" />
+  </node>
+
+  <node name="rviz" pkg="rviz" type="rviz" required="true"
+      args="-d $(find cartographer_ros)/configuration_files/demo_2d.rviz" />
+  <node name="playbag" pkg="rosbag" type="play"
+      args="--clock -r 2 $(arg bag_filename)" />
```
Comment (wohe) [inline cartographer_ros/launch/demo_revo_lds.launch]: Remove the -r 2?

**Comment by carl1218** [comment]
hi ，
 Can i run it online mode? not by loading a bag file.

**Comment by damonkohler** [comment]
Assuming you can find a ROS driver for the robot, of course. We are not providing one.

**Comment by Sumyer** [comment]
@damonkohler 
I checked that there is no 'tf' topic in the bag.
I wonder how cartographer work?
Thanks in advance!

**Comment by Sumyer** [comment]
I see that there is only one topic in the bag, that is the laser topic, and it owns frame_id.
But I still confuse that how I can evaluate the trajectory of this bag.

**Comment by Sumyer** [comment]
I get it.
Thanks all.

**Pull request merged**